libsolidity/ast/Types.h

```diff
@@ -344,6 +344,91 @@ class IntegerConstantType: public Type
 };
 
 /**
+ * Fixed data type, compromised of M bits on left and N bits on right side of decimal.
```

Comment:
Please use the term `Fixed point` and use the term `radix point` in favour of `decimal` (because we use binary numbers here). The class should also be name `FixedPointType`.